aggregator/flush_times_mgr.go

```diff
@@ -366,3 +373,12 @@ func (sc flushTimesChecker) HasFlushed(
 	sc.metrics.allFlushed.Inc(1)
 	return true
 }
+
+func standardFullyFlushed(flushtimes map[int64]int64, targetNanos int64) bool {
```

Comment:
Should this just be called `fullyFlushed` since it's used for timed metrics as well?

Reply:
sure